doc/source/configuration/firewall.rst

```diff
@@ -0,0 +1,350 @@
+.. _firewall:
+
+========
+Firewall
+========
+
+StackHPC Kayobe configuration provides a standardised firewalld configuration.
+The configuration uses the :kayobe-doc:`firewall
+<configuration/reference/hosts.html#firewalld>` host configuration
+functionality of Kayobe.
+
+The firewall configuration is provided in
+``etc/kayobe/inventory/group_vars/all/firewall``.
+
+Enabling StackHPC firewalld rules
+=================================
+
+The standardised firewalld configuration is not used by default and must be
+actively opted into. This can be done as follows:
+
+Controller firewalld Configuration
```

Comment:
```suggestion
Controller firewalld Configuration - ``controllers.yml``
```

Reply:
This is the title of the subsection. I don't think it needs the path in it since there's already a caption on the code block